### PR TITLE
fix 8025 solution

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 1]
```diff
@@ -159,7 +159,8 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
 
 **解决办法**:
 
-目前无法绕过 TiDB 的限制，只能忽略这张表，确保其它表顺利导入。
+- 使用系统变量 `tidb_txn_entry_size_limit` 动态增加限制。
+- 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 raft-entry-max-size，默认为 8MB，TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
```
Comment (hfxsd) [inline tidb-lightning/troubleshoot-tidb-lightning.md:163]:
```suggestion
- 使用系统变量 [`tidb_txn_entry_size_limit`](/stable/system-variables.md#tidb_txn_entry_size_limit-从-v760-版本开始引入) 动态增加限制。
- 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)（默认值为 `8MiB`），TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 2]
```diff
@@ -159,7 +159,8 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
 
 **解决办法**:
 
-目前无法绕过 TiDB 的限制，只能忽略这张表，确保其它表顺利导入。
+- 使用系统变量 [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-从-v760-版本开始引入) 动态增加限制。
+- 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)（默认值为 `8MiB`），TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
```
Comment (qiancai) [inline tidb-lightning/troubleshoot-tidb-lightning.md:163]:
```suggestion
- 注意，TiKV 也有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)（默认值为 `8MiB`），TiKV 会拒绝处理该请求。当表中单行的数据量较大时，需要同时修改这两个配置。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**